laravel-app/app/Http/Controllers/PostController.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Http\Requests\UserPostRequest;
+use App\Models\UsersPost;
+
+class PostController extends Controller
+{
+    public function index()
+    {
+        return view('post.post');
```

Comment:
hmmm?? can we name it post.index? why post.post? you can name the file of the post/index.blade.php

Reply:
okay sir copy that po, changing it now 🙇